service/src/test/java/uk/gov/hmcts/reform/fpl/service/ConfidentialDetailsServiceTest.java

```diff
@@ -448,75 +447,6 @@ void shouldNotRemoveConfidentialDetailsWhenNotMarkedAsConfidential() {
             assertThat(service.removeConfidentialDetails(others)).isEqualTo(others);
         }
 
-        @Test
-        void shouldReturnOthersIfOthersIsPrePopulated() {
-            CaseData caseData = CaseData.builder()
-                .others(Others.builder().firstOther(otherWithRemovedConfidentialFields(ID).getValue()).build())
-                .build();
-
-            List<Element<Other>> others = service.combineOtherDetails(caseData.getAllOthers(),
-                caseData.getConfidentialOthers());
-
-            assertThat(others.get(0).getValue()).isEqualTo(otherWithRemovedConfidentialFields(ID).getValue());
-        }
-
-        @Test
-        void shouldPrepareOtherWithConfidentialValuesWhenConfidentialOthersIsNotEmpty() {
-            CaseData caseData = CaseData.builder()
-                .others(Others.builder().firstOther(otherWithRemovedConfidentialFields(ID).getValue()).build())
-                .confidentialOthers(List.of(otherWithConfidentialFields(ID, CONFIDENTIAL)))
-                .build();
-
-            List<Element<Other>> others = service.combineOtherDetails(caseData.getAllOthers(),
-                caseData.getConfidentialOthers());
-
-            assertThat(others.get(0).getValue()).isEqualTo(otherWithConfidentialFields(ID, CONFIDENTIAL).getValue());
-        }
-
-        @Test
-        void shouldAddExpectedOtherWhenHiddenDetailsMarkedAsNo() {
-            CaseData caseData = CaseData.builder()
-                .others(Others.builder()
-                    .firstOther(otherWithConfidentialFields(ID, NOT_CONFIDENTIAL).getValue())
-                    .build())
-                .confidentialOthers(List.of(otherWithConfidentialFields(ID, CONFIDENTIAL)))
-                .build();
-
-            List<Element<Other>> others = service.combineOtherDetails(caseData.getAllOthers(),
-                caseData.getConfidentialOthers());
-
-            assertThat(others.get(0).getValue())
-                .isEqualTo(otherWithConfidentialFields(ID, NOT_CONFIDENTIAL).getValue());
-        }
-
-        @Test
-        void shouldMaintainOrderingOfOthersWhenComplexScenario() {
-            UUID otherId = randomUUID();
```

Comment:
Did we want to add similar test like this to the othersService? I think should be the same sort of logic except calling prepareOthers instesd of combineOtherDetails

Reply:
done